### PR TITLE
adds our couch.

### PR DESCRIPTION
**Pull request by kevinmcalear**

### CODE REVIEW

**Comment by captn3m0** [comment]
I am not sure how the h3 would render on the homepage. Will check it out locally to see if its good and then merge.

File: _couches/kevin&sarah.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+github: kevinmcalear
+twitter: kevinmcalear
+email: kevin.mcalear@gmail.com
+website: http://kevinmcalear.com/
+facebook: KPMcAlear
+city: Seattle
+country: USA
```
Comment (simison) [inline _couches/kevin&sarah.md:8]: Could you change `USA` to `US` before we'll merge this? Thanks!

Comment (captn3m0) [inline _couches/kevin&sarah.md:8]: Just realized another issue might be with the filename. /couches/kevin&sarah.html might be troublesome?

Let me check it out locally.

Comment (simison) [inline _couches/kevin&sarah.md:8]: Might work now but just to avoid issues in the future, I'd change it to `_and_`

Comment (captn3m0) [inline _couches/kevin&sarah.md:8]: Changed both country and filename before merging

**Pull request merged**